controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/ApplicationSerializer.java

```diff
@@ -514,7 +516,25 @@ private Change changeFromSlime(Inspector object) {
                                                 Instant.ofEpochMilli(object.field(atField).asLong())));
     }
 
-    private Optional<RotationId> rotationFromSlime(Inspector field) {
+    private List<RotationId> rotationsFromSlime(Inspector root) {
+        final var rotations = rotationListFromSlime(root.field(rotationsField));
+        final var legacyRotation = legacyRotationFromSlime(root.field(deprecatedRotationField));
```

Comment:
Consider adding a TODO here so that we remember to remove this after the next version has been released.

Reply:
Done.